packages/react-magma-dom/src/components/Stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import { Button } from '../Button';
+import { Stepper, StepperProps, Step, StepperLayout } from './';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import { Container } from '../Container';
+import { ButtonGroup } from '../ButtonGroup';
+import { InputType } from '../InputBase';
+import { Input } from '../Input';
+import { Flex, FlexAlignItems, FlexBehavior, FlexJustify } from '../Flex';
+import { LabelPosition } from '../Label';
+
+export default {
+  title: 'Stepper',
+  component: Stepper,
+  decorators: [
+    (Story, context) => (
+      <Container isInverse={context.args.isInverse} style={{ padding: '20px' }}>
+        <Story />
+      </Container>
+    ),
+  ],
+  argTypes: {
+    breakpoint: {
+      control: {
+        type: 'number',
+      },
+    },
+    breakpointLayout: {
+      control: {
+        type: 'select',
+        options: StepperLayout,
+      },
+    },
+    layout: {
+      control: {
+        type: 'select',
+        options: StepperLayout,
+      },
+    },
+    completionLabel: {
+      control: {
+        type: 'text',
+      },
+    },
+    stepLabel: {
+      control: {
+        type: 'text',
+      },
+    },
+    isInverse: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    testId: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
+} as Meta;
+
+const Template: Story<StepperProps> = args => {
+  const [currentStep, setCurrentStep] = React.useState(0);
+
+  const [numberOfSteps, setNumberOfSteps] = React.useState(4);
+
+  const handleOnNext = () => {
+    if (currentStep <= numberOfSteps) {
+      setCurrentStep(currentStep + 1);
+    }
+  };
+  const handleOnPrevious = () => {
+    if (currentStep !== 0) {
+      setCurrentStep(currentStep - 1);
+    }
+  };
+  const handleFinish = () => {
+    if (currentStep === numberOfSteps) {
+      setCurrentStep(0);
+    }
+  };
+
+  const handleOnChange = event => {
+    if (numberOfSteps > 0) {
+      return setNumberOfSteps(Number(event.target.value));
+    } else {
+      return setNumberOfSteps(1);
+    }
+  };
+
+  const steps = numberOfSteps;
+
+  const step = [...Array(steps)].map((_, i) => {
+    ++i;
+    return (
+      <Step
+        key={i}
+        testId={`Step ${i}`}
+        label={`Step ${i}`}
+        secondaryLabel={`Description area in secondaryLabel component ${i}`}
+      />
+    );
+  });
+
+  return (
+    <>
+      <Stepper currentStep={currentStep} {...args}>
```

Comment:
Lets add `ariaLabel` to our storybook example in case Perkins reviews this again from here